base/markdown/Common/inline.jl

```diff
@@ -30,14 +30,20 @@ end
 
 @trigger '`' ->
 function inline_code(stream::IO, md::MD)
-    result = parse_inline_wrapper(stream, "`"; rep=true)
-    return result === nothing ? nothing : Code(result)
-end
-
-@trigger '`' ->
-function inline_tex(stream::IO, md::MD)
-    result = parse_inline_wrapper(stream, "``"; rep=true)
-    return result === nothing ? nothing : LaTeX(result)
+    withstream(stream) do
+        ticks = startswith(stream, r"^(`+)")
+        result = readuntil(stream, ticks)
```

Comment:
I'm confused by this code.    `startswith` doesn't seem to have any methods that take an `IO` argument, and even if it did it seems like you're passing a boolean value to `readuntil`